packages/contracts/scripts/osx-versions-aliases.ts

```diff
@@ -7,5 +7,5 @@
  */
 export const OSX_VERSION_ALIASES = [
   '@aragon/osx-v1.0.1/',
-  '@aragon/osx-v1.3.0-rc0.4/',
+  '@aragon/osx-v1.3.0/',
```

Comment:
@mathewmeconry This should have happened in your PR too https://github.com/aragon/osx/pull/509.
Maybe we must bring this change into main.

Reply:
Nevermind, we don't have these integration tests on the `main` branch yet. They will end up there once we merge `develop` into `main` (probably when we publish `1.4.0`).